src/interfaces/isbn-registry/publishers.js

```diff
@@ -812,6 +813,15 @@ export default function () {
       }
     });
 
+    // Save audit entry
+    await auditEntryModel.create({
```

Comment:
Does this need await? Can't it just run on another thread?

Reply:
Nope, we want to know if the write fails before approving the download.